Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -14,14 +14,18 @@ public class Game {
 	}
 
 	public func playGame() {
+		let maxNumberOfGuesses = guessManager.maxNumberOfGuesses
+		view.promptNumberOfGuesses()
+		let numberOfGuesses = view.receiveNumberOfGuesses(maxNumberOfGuesses)
```

Comment:
I am not really sure how to separate this handling from the game class because it is within my game loop that the request for the number of guesses happens. My `Game` has access to my `View`, where the request is happening, but my guessManager does not have access to my view, so I cannot ask from there unless I pass in my instance of `View` from my `Game` to my `GuessManager`. 

I was writing the handling of the `numberOfGuesses` in the mindframe of handling the `guess`, which is still handled by the `GuessManager`, but received in the context of the game loop, and therefor passed into the `GuessManager` as an argument when needed from the `Game`